Pipfile

```diff
@@ -24,8 +24,9 @@ pylint = "<2"
 hypothesis = "*"
 sphinx = "*"
 sphinxcontrib-websupport = "*"
-sphinx-autodoc-typehints = "*"
+sphinx-autodoc-typehints = "==1.8.0"
 astroid = "<2.2"
+mypy-extensions = "==0.4.2"
```

Comment:
It's unclear to me how this and the corresponding hash in `Pipfile.lock` got in a state causing the builds to fail, but seems fixed now...